[rospy.client][INFO] 2019-01-14 09:56:08,440: init_node, name[/rostopic_4656_1547466968366], pid[4656]
[xmlrpc][INFO] 2019-01-14 09:56:08,441: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-01-14 09:56:08,441: Started XML-RPC server [http://onboardpc:33349/]
[rospy.init][INFO] 2019-01-14 09:56:08,442: ROS Slave URI: [http://onboardpc:33349/]
[rospy.impl.masterslave][INFO] 2019-01-14 09:56:08,442: _ready: http://onboardpc:33349/
[xmlrpc][INFO] 2019-01-14 09:56:08,442: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-01-14 09:56:08,442: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-01-14 09:56:08,542: registered with master
[rospy.rosout][INFO] 2019-01-14 09:56:08,543: initializing /rosout core topic
[rospy.rosout][INFO] 2019-01-14 09:56:08,547: connected to core topic /rosout
[rospy.simtime][INFO] 2019-01-14 09:56:08,550: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2019-01-14 09:56:08,609: topic[/tf] adding connection to [http://onboardpc:44841/], count 0
[rospy.internal][INFO] 2019-01-14 09:56:08,830: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2019-01-14 09:56:15,212: signal_shutdown [signal-2]
[rospy.internal][INFO] 2019-01-14 09:56:15,219: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2019-01-14 09:56:15,219: topic[/tf] removing connection to http://onboardpc:44841/
[rospy.topics][ERROR] 2019-01-14 09:56:15,222: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.internal][WARNING] 2019-01-14 09:56:15,222: Unknown error initiating TCP/IP socket to onboardpc:58393 (http://onboardpc:44841/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.impl.masterslave][INFO] 2019-01-14 09:56:15,222: signal-2
[rospy.core][INFO] 2019-01-14 09:56:15,331: signal_shutdown [atexit]
